### ISSUE TITLE
SSL method loss

### ISSUE THREAD

**Issue by dinzhenofficial** [betterzhou/DECL#4]
@betterzhou   

In this paper's auto-noise reduction selection part, you propose combining autoregressive and comparative learning joint loss for auto-noise reduction selection. Still, the autoregressive loss in this part of your paper seems to be calculated differently that in your paper. Where is this a problem?

`loss = (temp_cont_loss0 + temp_cont_loss_noisy) / 2 * lambda1 + denoise_objective`
